analyzers/tests/SonarAnalyzer.UnitTest/Helpers/ProjectTypeHelperTests.cs

```diff
@@ -63,34 +63,34 @@ public void TestReference_ShouldBeSynchronized()
         [TestMethod]
         public void IsTest_ReturnsTrueForTestFrameworks()
         {
-            CreateContext(NuGetMetadataReference.JetBrainsDotMemoryUnit(Constants.NuGetLatestVersion)).IsTest().Should().BeTrue();
-            CreateContext(NuGetMetadataReference.MSTestTestFrameworkV1).IsTest().Should().BeTrue();
-            CreateContext(NuGetMetadataReference.MSTestTestFramework(Constants.NuGetLatestVersion)).IsTest().Should().BeTrue();
-            CreateContext(NuGetMetadataReference.MicrosoftVisualStudioQualityToolsUnitTestFramework).IsTest().Should().BeTrue();
-            CreateContext(NuGetMetadataReference.MachineSpecifications(Constants.NuGetLatestVersion)).IsTest().Should().BeTrue();
-            CreateContext(NuGetMetadataReference.NUnit(Constants.NuGetLatestVersion)).IsTest().Should().BeTrue();
-            CreateContext(NuGetMetadataReference.NUnitLite(Constants.NuGetLatestVersion)).IsTest().Should().BeTrue();
-            CreateContext(NuGetMetadataReference.SpecFlow(Constants.NuGetLatestVersion)).IsTest().Should().BeTrue();
-            CreateContext(NuGetMetadataReference.XunitFrameworkV1).IsTest().Should().BeTrue();
-            CreateContext(NuGetMetadataReference.XunitFramework(Constants.NuGetLatestVersion)).IsTest().Should().BeTrue();
+            CreateSemanticModel(NuGetMetadataReference.JetBrainsDotMemoryUnit(Constants.NuGetLatestVersion)).Compilation.IsTest().Should().BeTrue();
```

Comment:
_To consider:_ As there's only one signature now, updating it like this will make 20 lines 30 characters shorter.
```suggestion
            IsTest(NuGetMetadataReference.JetBrainsDotMemoryUnit(Constants.NuGetLatestVersion)).Should().BeTrue();
```